tests/io/check_connectivity/connectivity_test.cpp

```diff
@@ -62,6 +63,18 @@ class ConnectivityCheckSuite : public ::testing::Test {
       // Third Patch
       Point2D{1., 0.}, Point2D{2., 0.}, Point2D{2., 1.}, Point2D{1., 1.}};
 
+  // Face center points embdedded
+  std::vector<Point3D> corner_vertices_3d{
+      // First Patch
+      Point3D{0., 1., 2.}, Point3D{1., 1., 2.}, Point3D{1., 2., 2.},
+      Point3D{0., 2., 2.},
+      // Second Patch
+      Point3D{1., 1., 2.}, Point3D{2., 1., 2.}, Point3D{2., 2., 2.},
+      Point3D{1., 2., 2.},
+      // Third Patch
+      Point3D{1., 0., 2.}, Point3D{2., 0., 2.}, Point3D{2., 1., 2.},
+      Point3D{1., 1., 2.}};
+
   // Expected Connectivity
   std::vector<std::array<std::size_t, 4>> expected_connectivity{
```

Comment:
can you explain this?

Reply:
`static_cast<std::size_t>(-1)` ?

Reply:
Ah I thought you were referring to the lines above. "negativ" values just mean that there is a boundary, the abs value is the boundary ID